content/docs/studio/get-started.md

```diff
@@ -1,31 +1,38 @@
 # Get Started with DVC Studio
 
 Here, we will walk you through a tutorial to use DVC Studio for collaboration on
-your ML projects. You will need access to a Github, Gitlab or Bitbucket account
+your ML projects. You will need access to a GitHub, GitLab or Bitbucket account
 which has access to the Git repositories you want to connect. DVC Studio creates
 views from repositories when you connect to them.
 
+The following video provides you a quick overview of DVC Studio.
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
 ## What is a view?
 
 A _view_ is an interactive representation of the experiments run in your ML
 project. DVC Studio automatically identifies datasets, metrics and
```

Comment:
I think `automatically` is not the best term here. It doesn't do anything automatically. It parses Git history and specific metafiles (or files you show to it). Automatically sounds like some magic is happening.

Reply:
removed

Reply:
"intelligently?"